src/main/java/org/worldcubeassociation/workbook/wcajson/WcaCompetitionJson.java

```diff
@@ -9,4 +9,5 @@
 	public WcaPersonJson[] persons;
 	public WcaEventJson[] events;
 	public String scrambleProgram;
+	public String resultsProgram;

```

Comment:
Wherever we fill in `formatVersion`, we need to specify version 0.3 of the wca json format, not 0.2.